src/ontology/mondo-ingest.Makefile

```diff
@@ -563,12 +563,18 @@ $(EXTERNAL_CONTENT_DIR)/%.robot.owl: $(EXTERNAL_CONTENT_DIR)/%.robot.tsv
 $(EXTERNAL_CONTENT_DIR)/nord.robot.tsv: $(TMPDIR)/nord.tsv config/external-content-robot-headers.json
 	mkdir -p $(EXTERNAL_CONTENT_DIR)
 	python ../scripts/add-robot-template-header.py $^ > $@
-.PRECIOUS: $(EXTERNAL_CONTENT_DIR)/nord.robot.owl
+.PRECIOUS: $(EXTERNAL_CONTENT_DIR)/nord.robot.tsv
 
-.PHONY: external-content-nord
+$(EXTERNAL_CONTENT_DIR)/nando-mappings.robot.tsv: $(MAPPINGSDIR)/mondo-nando.sssom.tsv
```

Comment:
@matentzn FYI: had to flip filename. You had `nando-mondo.sssom.tsv` here, but the file that is committed is `mondo-nando.sssom.tsv`

Reply:
thx

Reply:
@joeflack4 does `nando-mondo.sssom.tsv` need to be removed from the commit?

Reply:
@twhetzel Good question. Originally, I believe there was just 1 file. I looked now and there is indeed a `mondo-nando` and a `nando-mondo` variation. This doesn't seem to be an error. The subject/objects correctly match with the naming order. And there is a separate goal for each of these in the `makefile`. So I guess I'm not worried about it, but it would be helpful if @matentzn could clarify why we're doing this.

Reply:
The nando-mondo mappings were supplied by the Nando, the pipeline invertes it in row 595 to be mondo-nando which is what we need in Mondo. Then you generate the ROBOT template from it in externally provided content! All looks great.